src/software/new_geom/util/physics.cpp

```diff
@@ -0,0 +1,23 @@
+#include "software/new_geom/util/physics.h"
+
+Point calculateFuturePosition(const Point &initial_position,
+                              const Vector &initial_velocity, const Vector &acceleration,
+                              double time_in_future)
+{
+    // Using second equation of motion from https://physics.info/motion-equations/
+    // ∆s = v0t + ½at2
```

Comment:
✨ 

Might be missing a ^ in your ^2 here

Reply:
fixed